### PR TITLE
Update apps_wishlist.md (BitcartCC, BTCPayServer, Commento++)

### PR DESCRIPTION
**Pull request by collector-ynh**
I have added the **BitcartCC** and **Commento++** applications in the Wishlist, and I have updated the description and the URL of the **BTCPayServer** application which is paramount for **YunoHost**.

### CODE REVIEW

**Comment by ericgaspar** [comment]
Commento is already part of the catalog https://github.com/YunoHost-Apps/commento_ynh

File: pages/02.applications/04.wishlist/apps_wishlist.md [hunk 1]
```diff
@@ -34,11 +34,12 @@ You can [contribute to this list by adding something you'd like to be packaged](
 | [Baserow](https://baserow.io/) | Open source no-code database tool and Airtable alternative. | [Upstream](https://gitlab.com/bramw/baserow) |  |
 | [Beeper](https://www.beeper.com/) | A unified inbox for 15 chat networks. | [Upstream](https://gitlab.com/beeper) |  |
 | [BigBlueButton](https://bigbluebutton.org) | Web conferencing system | [Upstream](https://github.com/bigbluebutton/bigbluebutton) |  |
+| [BitcardCC](https://bitcartcc.com) | BitcartCC is a self-hosted, open-source cryptocurrency all-in-one solution. It's secure, private, censorship-resistant and free | [Upstream](https://github.com/bitcartcc/bitcart)  |  |
```
Comment (MrNaif2018) [inline pages/02.applications/04.wishlist/apps_wishlist.md:37]:
```suggestion
| [BitcartCC](https://bitcartcc.com) | BitcartCC is a self-hosted, open-source cryptocurrency all-in-one solution. It's secure, private, censorship-resistant and free | [Upstream](https://github.com/bitcartcc/bitcart)  |  |
```
(:

**Comment by MrNaif2018** [comment]
By the way, actually this can be out of wishlist. How could I help BitcartCC to be added to YunoHost?

File: pages/02.applications/04.wishlist/apps_wishlist.md [hunk 2]
```diff
@@ -52,6 +53,7 @@ You can [contribute to this list by adding something you'd like to be packaged](
 | [CloudTube](https://tube.cadence.moe/) | CloudTube front-end for YouTube | [Upstream](https://git.sr.ht/~cadence/cloudtube) |  |
 | [Cockpit](https://cockpit-project.org/) |  |  | [Package Draft](https://github.com/YunoHost-Apps/cockpit_ynh) |
 | coin | Member dashboard for non profit ISP | [Upstream](https://code.ffdn.org/FFDN/coin/) | [Package Draft](https://github.com/YunoHost-Apps/coin_ynh) |
+| CommentoPlusPlus | Commento++ allows you to foster discussion on your website | [Upstream](https://github.com/souramoo/commentoplusplus) | [Pakage Draft](https://github.com/YunoHost-Apps/commento_ynh/tree/plusplus) |
```
Comment (ericgaspar) [inline pages/02.applications/04.wishlist/apps_wishlist.md:56]:
```suggestion
```

File: pages/02.applications/04.wishlist/apps_wishlist.md [hunk 3]
```diff
@@ -34,11 +34,12 @@ You can [contribute to this list by adding something you'd like to be packaged](
...
 | [BigBlueButton](https://bigbluebutton.org) | Web conferencing system | [Upstream](https://github.com/bigbluebutton/bigbluebutton) |  |
+| [BitcardCC](https://bitcartcc.com) | BitcartCC is a self-hosted, open-source cryptocurrency all-in-one solution. It's secure, private, censorship-resistant and free | [Upstream](https://github.com/bitcartcc/bitcart)  |  |
 | [Bitmessage](https://bitmessage.org/) | P2P communication protocol used to send encrypted messages | [Upstream](https://github.com/Bitmessage/PyBitmessage) |  |
 | Blynk |  | [Upstream](https://github.com/blynkkk/blynk-library) |  |
 | [Bolt](https://bolt.cm/) | Content Management Tool |  | [Package Draft](https://github.com/realitygaps/bolt_ynh) |
 | BookWyrm | Social reading and reviewing, decentralized with ActivityPub | [Upstream](https://github.com/mouse-reeve/bookwyrm) |  |
-| BTCPay Server |  | [Upstream](https://github.com/btcpayserver/btcpayserver) |  |
+| [BTCPay Server](https://btcpayserver.org) | BTCPay Server is a free and open-source Bitcoin payment processor which allows you to accept bitcoin without fees or intermediaries | [Upstream](https://github.com/btcpayserver/btcpayserver) |  |
```
Comment (ericgaspar) [inline pages/02.applications/04.wishlist/apps_wishlist.md:42]:
```suggestion
| [BTCPay Server](https://btcpayserver.org) | BTCPay Server is a free and open-source Bitcoin payment processor | [Upstream](https://github.com/btcpayserver/btcpayserver) |  |
```

**Review by ericgaspar — APPROVED** [review]

**Pull request merged**